find_changed_files.sh

```diff
@@ -37,14 +45,14 @@ then
         exit 1
     fi
     # TODO paginate to support more than 100 files
-    COMPARE_RESPONSE=$(curl -H "Accept: application/vnd.github.v3+json" "$COMPARE_API?per_page=100")
+    COMPARE_RESPONSE=$(curl -H $FORMAT_HEADER -H "$TOKEN_HEADER" "$COMPARE_API?per_page=100")
```

Comment:
Does this actually work if we don't pass GITHUB_TOKEN since it's not a required arg?

Do we end up with `curl -H $FORMAT_HEADER -H "" "$COMPARE_API?per_page=100"` or does the blank header not matter?

What about just:

```
if [ -z "$GITHUB_TOKEN" ]; then
  FILES_JSON=$(curl -H $FORMAT_HEADER -H "$TOKEN_HEADER" "$PR_FILES_API?per_page=100")
else
  FILES_JSON=$(curl -H $FORMAT_HEADER "$PR_FILES_API?per_page=100")
fi
```

Reply:
This is also POSIX-compliant :d

Reply:
Also how come we don't just append ?per_page=100 to `PR_FILES_API`?

Reply:
yea blank header is just no-op.
i'll merge this for now so things start working.

i had some problems with format parameters though i was building the whole header line, maybe separating could help but i need to have two versions so it becomes two if checks. this works fine :)